lib/cli/stats.js

```diff
@@ -246,8 +280,16 @@ async function getTemplatesSummary() {
     summary.sharedParts.externallyManaged +
     summary.exportFiles.externallyManaged +
     summary.accountTemplates.externallyManaged;
+  summary.all.externallyManagedPerc = percentageRoundTwo(
+    summary.all.externallyManaged,
+    summary.all.total
+  )
   summary.all.yamlFiles =
     summary.reconciliations.yamlFiles + summary.accountTemplates.yamlFiles;
+  summary.all.yamlFilesPerc = percentageRoundTwo(
+    summary.all.yamlFiles,
+    summary.all.total
```

Comment:
[TODO] Should we exclude shared parts (and possibly export files) from this total figure as _cannot_ have YAML tests for these items. Could also mean we need a separate total 'testable' value

Reply:
Good catch. Updated to only consider RT and ADT